### ISSUE TITLE
Errors after update

### ISSUE THREAD

**Issue by simoneldevig** [robbyrussell/oh-my-zsh#6764]
After updating two days ago I get the following erros every time I open my bash.

```
/home/simoneldevig/.oh-my-zsh/lib/git.zsh:42: parse error near `elif'
/home/simoneldevig/.oh-my-zsh/lib/history.zsh:6: command not found: ^M
/home/simoneldevig/.oh-my-zsh/lib/history.zsh:11: parse error near `elif'
zsh: command not found: git_prompt_info
```

Just tried to update again after reading #6758 but the problem still presists.

Running Ubuntu bash on Windows 10.

Any suggestions on how to fix it?

**Comment by mcornella** [comment]
Another issue that _might_ be fixed by #4071.

**Comment by onedou** [comment]
thanks @mcornella 
It's works well
![image](https://user-images.githubusercontent.com/3422640/42354347-d1364d7e-80f8-11e8-91e8-43dbdf7939aa.png)
